src/TBuffer.h

```diff
@@ -149,7 +149,7 @@ class TBuffer
     TBuffer cut(QPoint&, QPoint&);
     void paste(QPoint&, TBuffer);
     void setBufferSize(int s, int batch);
-    static const QList<QString> getComputerEncodingNames();
+    static const QList<QString> getComputerEncodingNames() { return csmEncodingTable.keys(); };
```

Comment:
😕 Is it really both `static` and `const` - normally a `const` on a function return is useless unless it is a reference that is being passed and `QMap<T1,T2>::keys()` returns `QList<T1>` as a value __not__ a reference. Though `csmEncodingTable` __is__ a `static`, `const` item...